Formula/at-spi2-core.rb

```diff
@@ -10,11 +10,10 @@ class AtSpi2Core < Formula
   end
 
   depends_on "gobject-introspection" => :build
-  depends_on "intltool" => :build
```

Comment:
Looks like it was dropped a while back https://gitlab.gnome.org/GNOME/at-spi2-core/-/commit/154ca0e62a4d8581b1a4bdf6fe556cd5e1e0aed8